src/rules/__tests__/no-try-expect.test.js

```diff
@@ -0,0 +1,66 @@
+import { RuleTester } from 'eslint';
+import rule from '../no-try-expect';
+
+const ruleTester = new RuleTester({
+  parser: require.resolve('babel-eslint'),
+});
+
+const errors = [
+  {
+    messageId: 'noTryExpect',
+  },
+];
+
+ruleTester.run('no-try-catch', rule, {
+  valid: [
+    {
+      code: `it('foo', () => {
```

Comment:
Typically we omit the braces in these tests when they're not needed.

Reply:
TIL!